docs/source/settings.rst

```diff
@@ -609,6 +609,16 @@ The Access log file to write to.
 
 ``'-'`` means log to stdout.
 
+.. _disable-access-log-redirection:
+
+disable_access_log_redirection
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+* ``--disable-access-log-redirection``
+* ``False``
+
+Disable redirect access logs to syslog.
+
```

Comment:
Also, it would be nice to add a note to `syslog` documentation at http://docs.gunicorn.org/en/stable/settings.html#syslog

```rst
.. versionchanged:: 19.8

   You can now disable sending access logs by using the
   :ref:`disable-access-log-redirection` setting.
```